src/components/modal/index.tsx

```diff
@@ -74,11 +74,11 @@ const Modal: FC<ModalProps> = ({
           <CloseMIcon color={style === "dark" ? "#FFFFFF" : "#232A36"} />
         </div>
         <div
-          className={classNames("z-modal w-12/12 my-0", {
-            "p-40 md:w-modalLarge": size === "large",
-            "p-40 md:w-modal": size === "medium",
-            "p-40 md:w-modalSmall": size === "small",
-            "h-full": size === "fullscreen",
+          className={classNames("z-modal w-12/12 my-0 p-15", {
```

Comment:
Did you verify with the designer that it's fine to change the default padding for every modal on mobile?

Reply:
Yes, he noticed that the padding is too big on mobile